title.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>DanBot Docs</title>
```

Comment:
This should also probably say “Danbot Hosting Docs” or “DBH Docs” as thats how we always have it say

Reply:
> This should also probably say “Danbot Hosting Docs” or “DBH Docs” as thats how we always have it say

Dan needs to start cracking down on those that say "Danbot" instead of "DanBot." And yes, the capitalization of that "b" absolutely matters.

Reply:
It really doesn’t.